sass/module/_availability.scss

```diff
@@ -13,6 +13,7 @@
   color: darken($availability-reservable, 10%);
 }
 
+#user-bookmarks .reserve-button,
```

Comment:
You'd think so... however, css/ddbasic.styles.css define .pane-bookmarks .action-button as display:inline-block, and it has a higher weight than .material-item .reserve-button
Alternatively, .pane-bookmarks .action-button.reserve-button could be used as a selector?